src/main/java/org/sonatype/nexus/repository/puppet/internal/PuppetRecipeSupport.groovy

```diff
@@ -20,7 +20,7 @@ import org.sonatype.nexus.repository.http.PartialFetchHandler
 import org.sonatype.nexus.repository.httpclient.HttpClientFacet
 import org.sonatype.nexus.repository.puppet.internal.security.PuppetSecurityFacet
 import org.sonatype.nexus.repository.purge.PurgeUnusedFacet
-import org.sonatype.nexus.repository.search.SearchFacet
+import org.sonatype.nexus.repository.search.ElasticSearchFacet
```

Comment:
I'm no expert, but I saw some comment from other formats where it was better to use "org.sonatype.nexus.repository.search.index.SearchIndexFacet" instead of "org.sonatype.nexus.repository.search.ElasticSearchFacet". Could you give that a try and see if it works for you?

e.g.: https://github.com/sonatype-nexus-community/nexus-repository-composer/commit/8bb9662b36afc79a36972883f25defb85b540fde

Reply:
I'll try...

Reply:
Works. Code updated. Thx for review!